speiseplan_projekt/src/app/api/checkAdmin/route.js

```diff
@@ -0,0 +1,17 @@
+import { verifyAuth } from '@/lib/verifyToken';
+import { NextResponse } from 'next/server';
+
+export async function GET(req) {
+    const token = req.cookies.get('token')?.value;
+
+    if (!token) {
+        return NextResponse.json({ isAdmin: false }, { status: 401 });
+    }
+
+    try {
+        const verifiedToken = await verifyAuth(token);
+        return NextResponse.json({ isAdmin: verifiedToken.admin }, { status: 200 });
+    } catch (err) {
+        return NextResponse.json({ isAdmin: false }, { status: 500 });
+    }
```

Comment:
benutz die verify admin function in lib/verifyToken.ts die gibt wenn valid den token payload zurueck

Reply:
kann ichs nicht so lassen sonst muss ich alles wieder umbauen...

Reply:
Passt des wenn wirs so lassen, weil des hat über 3h gedauert bis ichs so hatte und ist es au ned besser des im backend zu machen wegen sicherheit

Reply:
passt war nur ein winziger change

Reply:
asooo jetzt weis ich was du meintest

perfekt danke xD